app/utils/simplified-account/format/format-card-types.test.js

```diff
@@ -0,0 +1,111 @@
+const { expect } = require('chai')
+const { formatCardTypesForTemplate } = require('@utils/simplified-account/format/format-card-types')
+
+const allCards = [
+  {
+    id: 'id-001',
+    brand: 'visa',
+    label: 'Visa',
+    type: 'DEBIT',
+    requires3ds: false
+  },
+  {
+    id: 'id-002',
+    brand: 'visa',
+    label: 'Visa',
+    type: 'CREDIT',
+    requires3ds: false
+  },
+  {
+    id: 'id-003',
+    brand: 'master-card',
+    label: 'Mastercard',
+    type: 'DEBIT',
+    requires3ds: false
+  },
+  {
+    id: 'id-004',
+    brand: 'american-express',
+    label: 'American Express',
+    type: 'CREDIT',
+    requires3ds: false
+  },
+  {
+    id: 'id-005',
+    brand: 'jcb',
+    label: 'Jcb',
+    type: 'CREDIT',
+    requires3ds: false
+  },
+  {
+    id: 'id-006',
+    brand: 'maestro',
+    label: 'Maestro',
+    type: 'DEBIT',
+    requires3ds: true
+  }
+]
+describe('format-card-types for template', () => {
```

Comment:
Testing for these [hints](https://github.com/alphagov/pay-selfservice/pull/4376/files#diff-7ab8e2610a16c7fd8d73819249d18f2678d1d4e216024df5d39f519918bf52fbR22-R23) in this test would be good too?

Reply:
good point - this is now done.